File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/sqlite/RealTypes.scala
package dev.typr.foundationssc.docs.sqlite
import dev.typr.foundationssc.*

@SuppressWarnings(Array("unused"))
object RealTypes:
  // start
  val realType: SqliteType[Double] = SqliteTypes.real
  val doubleType: SqliteType[Double] = SqliteTypes.double_
  val floatType: SqliteType[Float] = SqliteTypes.float_
  // stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/sqlite/RealTypes.java
package dev.typr.foundations.docs.sqlite;

import dev.typr.foundations.SqliteType;
import dev.typr.foundations.SqliteTypes;

@SuppressWarnings("unused")
public class RealTypes {
  // start
  SqliteType<Double> realType = SqliteTypes.real; // REAL (canonical, 8-byte IEEE 754)
  SqliteType<Double> doubleType = SqliteTypes.double_;
  SqliteType<Float> floatType = SqliteTypes.float_;
  // stop
}
